Add Gatsby installation to startup docs (#904)
primer/components: CONTRIBUTING.md

We primarily use our documentation site as a workspace to develop new components or make changes to existing components (stay tuned for a better development environment coming soon!).

Before running the documentation site locally, you'll need to install packages in the root and `docs` directories:

```shell

> yarn && cd docs; yarn 

Author: ```suggestion
yarn && cd docs && yarn 
```

> Then run the following to start up the site:

Author: ```suggestion
Then navgiate back to the root folder and run the following to start up the site:
```

We primarily use our documentation site as a workspace to develop new components or make changes to existing components (stay tuned for a better development environment coming soon!).

> Before running the documentation site locally you'll need to install packages and make sure you've got the [Gatsby cli](https://www.gatsbyjs.com/docs/gatsby-cli/) installed on your machine:
> 
> ```sh
> yarn && yarn global add gatsby-cli
> ```

Reviewer: ```suggestion
Before running the documentation site locally, you'll need to install packages in the root and `docs` directories:

Gatsby is already a dependency in the `docs` site. I think the issue that caused #903 is that the `docs` dependencies weren't installed.
```shell
yarn && cd docs; yarn 
```
```
